packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -564,4 +564,102 @@ void main() {
     expect(outsideBoldTextOverride, false);
     expect(insideBoldTextOverride, true);
   });
+
+  testWidgets('MediaQueryData can not be initialized with null values', (WidgetTester tester) async {
+
+    String generateErrorMessage(String parameterName){
+      return '\'$parameterName != null\': is not true';
+    }
+
+    try{
+      MediaQueryData(size: null);
+    }on AssertionError catch(error){
+      expect(error.toString(), contains(generateErrorMessage('size')));
+    }
```

Comment:
For all the try/catch blocks:
```suggestion
    try {
      MediaQueryData(size: null);
    } on AssertionError catch (Exception error) {
      expect(error.toString(), contains(generateErrorMessage('size')));
    }
```